shell/common/node_bindings.cc

```diff
@@ -164,21 +164,23 @@ void SetNodeCliFlags() {
     const auto& option = arg;
 #endif
     const auto stripped = base::StringPiece(option).substr(0, option.find('='));
-    if (allowed.count(stripped) != 0)
+
+    // Only allow in no-op (--) option or DebugOptions
+    if (allowed.count(stripped) != 0 || stripped == "--")
```

Comment:
@codebytere why was this check for no-op separator become explicit in this PR ? Having this is good, just curious about the cause.

Reply:
@deepak1556 previous to this we were parsing the `DebugOptions` separately, so while we were stripping them in this function, it wasn't the results of this function that DebugOptions was seeing. This would have been wrong before, but we just weren't seeing the consequences of it until we used it for its intended purpose in this PR.